api/graphql/applications/application_types.py

```diff
@@ -107,16 +112,58 @@ class ApplicationEventAggregatedDataType(graphene.ObjectType):
     allocation_results_reservations_total = graphene.Float()
 
 
+class ApplicationEventScheduleResultType(AuthNode, PrimaryKeyObjectType):
+    permission_classes = (
+        (AllowAuthenticated,) if not settings.TMP_PERMISSIONS_DISABLED else [AllowAny]
+    )
+    allocated_reservation_unit = graphene.Field(ReservationUnitType)
+    basket = graphene.Field(ApplicationRoundBasketType)
+    allocated_day = graphene.Field(
+        graphene.Enum("allocatedDay", [(v.upper(), k) for k, v in DAY_CHOICES])
+    )
+
+    class Meta:
+        model = ApplicationEventScheduleResult
+        fields = (
+            "id",
+            "pk",
+            "accepted",
+            "declined",
```

Comment:
What is the reason we have separate fields for `accepted` and `declined`? I know this comes from `ApplicationEventScheduleResult` but do you know/remember the reason? 🤔

Reply:
declineds are for the weekly amount reduction logic for the algorithm